pkg/config/config.go

```diff
@@ -67,6 +67,13 @@ const (
 	// ServingNamespaceEnv is an env variable specifying where the serving is deployed.
 	// e.g. OpenShift deploys Kourier in different namespace so `system.Namespace()` does not work.
 	ServingNamespaceEnv = "SERVING_NAMESPACE"
+
+	// TenantNameLabelKey is the annotation key for grouping/isolating ingresses by tenant
+	TenantNameLabelKey = "admin.knative.dev/tenant-name"
```

Comment:
If this is being used to generate the listener service name, why not use the port number in the service name to avoid needing to correlate a second string (e.g. `kourier-isolation-6452` as a Service name, where the first part is a fixed string)?

Reply:
This is an excellent point! There is no need for the listener label anymore. `tenant-name` feels wrong in this context anyway so I'm more that happy to remove this label!